percy/snapshot.py

```diff
@@ -81,3 +81,28 @@ def percy_snapshot(driver, name, **kwargs):
     except Exception as e:
         print(f'{LABEL} Could not take DOM snapshot "{name}"')
         print(f'{LABEL} {e}')
+
+# Take screenshot on driver
+def percy_automate_screenshot(driver, name, **kwargs):
+    if not is_percy_enabled(): return
```

Comment:
@chinmay-browserstack  add a new check that driver should be only `selenium` session not an `appium` session